__tests__/api.test.js

```diff
@@ -0,0 +1,85 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+const { handleServerErrors } = require('../controllers/error-handling-controller');
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+
+describe('GET /api/topics', () => {
+    test('responds with a 200 status code', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    })
+    test('get /api/topics request responds with an array of topics objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(res => {
+            expect(res.body.topics).toBeInstanceOf(Array);
+            expect(res.body.topics).not.toBeEmpty();
+            expect(res.body.topics[0]).toHaveProperty('slug');
+            expect(res.body.topics[0]).toHaveProperty('description');
+            for (let i = 0; i < res.body.topics.length; i++) {
+                expect(res.body.topics[i]).toBeInstanceOf(Object);
+                expect(res.body.topics[i]).not.toBeEmpty();
+                expect(res.body.topics[i]).toHaveProperty('slug');
+                expect(res.body.topics[i]).toHaveProperty('description');
+            }
+        })
+    })
+})
+
+describe('GET /api/articles', () => {
+    test('responds with a 200 status code', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+    })
+    test('get /api/articles request responds with an array of articles objects with a comment_count property and date in decending order', () => {
```

Comment:
How is this testing for the descending order of the date?
An application of [jest-sorted](https://www.npmjs.com/package/jest-sorted) would be very useful here!